blueos_repository/registry.py

```diff
@@ -55,9 +55,7 @@ async def fetch_remote_tags(self, repository: str) -> List[str]:
                 data = await resp.json(content_type=None)
                 tags = data["results"]
 
-                valid_images = [tag["name"] for tag in tags]
-                print(valid_images)
```

Comment:
It would be nice if we could add a debug message like this one somewhere in the code